### DISCUSSION TITLE
Please upload a new nuget package of Serilog.Exceptions.EntityFrameworkCore

### DISCUSSION THREAD

**Discussion by TiborBremer** [RehanSaeed/Serilog.Exceptions#157] [Q&A]
At the moment the nuget version of Seriglog.Exceptions.EntityFrameworkCore is at 5.3.1 which is the version with the incorrect namespace. This is fixed in version 5.3.2 but this isn't on nuget.

I thought I had accidentally installed the wrong package when I needed the SQLServer namespace inside my project

**Comment by RehanSaeed** [comment]
Released. Thank you for pointing that out.
